blob/s3blob/s3blob_test.go

```diff
@@ -233,10 +239,16 @@ func (v verifyContentLanguage) BeforeList(as func(interface{}) bool) error {
 				return errors.New("List.As failed")
 			}
 		} else {
-			var req *s3v2.ListObjectsV2Input
-			if !as(&req) {
-				return errors.New("List.As failed")
+			var (
+				list *s3v2.ListObjectsV2Input
+				opts *[]func(*s3v2.Options)
+			)
+			// make sure we can extract ALL the underlying GCS types
```

Comment:
I think you can drop this comment and make the code look more like above, e.g.:

if !as(&list) || !as(&opts) {
  return errors.New(...)
}
return nil